### DISCUSSION TITLE
希望出一个窗口置顶的功能

### DISCUSSION THREAD

**Discussion by UJohn** [rxhanson/Rectangle#442] [Ideas]
窗口置顶最前的功能，防止被其他窗口遮住

**Comment by rxhanson** [comment]
See https://github.com/rxhanson/Rectangle/discussions/289 & https://github.com/rxhanson/Rectangle/discussions/280.

A better (IMO) alternative that already exists in Rectangle is Todo Mode: https://github.com/rxhanson/Rectangle/wiki/Todo-Mode
